package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "launchpad",
   "description": "You can launch browsers! From NodeJS! Local ones! Remote ones! Browserstack ones!",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
No need to manually bump this number, we have a release script: https://github.com/bitovi/launchpad/blob/91d7782e48395af9367246bd8da148a5a84cd155/package.json#L36